README.md

```diff
@@ -4,16 +4,107 @@
 [![Build Status](https://travis-ci.org/jboss-fuse/vscode-datavirt.svg?branch=master)](https://travis-ci.org/jboss-fuse/vscode-datavirt)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Prerequisites
+
+- recent Visual Studio Code
+
+
 ## Features
 
-...
+Red Hat Integration - Data Virtualization Tooling enables you to create a Virtual Database definition file (VDB) and to visualize and manipulate its contents inside [VS Code](https://code.visualstudio.com/).
```

Comment:
I think it would be convenient to link the marketplace entry from the name `Red Hat Integration - Data Virtualization Tooling`

Reply:
Is there a way to predict the URL if there hasn't been a release yet?

Reply:
yes, it is https://marketplace.visualstudio.com/items?itemName=<publichserID>.<extension-id>
so I think it would be https://marketplace.visualstudio.com/items?itemName=redhat.vscode-datavirt

for instance https://marketplace.visualstudio.com/items?itemName=redhat.vscode-apache-camel

but that's a good point that it is not yet published. Not sure what is the best:
- provide the link even if not working until release
- leave as-is for now and create an issue as a "post initial release task"

Reply:
This is not consistently done in our set of extensions. It might be good to do it here, but also create a JIRA somewhere so we can add the marketplace link to ALL of our extensions.